lib/rules/order-in-routes.js

```diff
@@ -10,6 +10,7 @@ const ORDER = [
   'service',
   'inherited-property',
   'property',
+  'init',
```

Comment:
same question as above

Reply:
> I'd suggest to remove init from the component lifecycle hooks as well.

@Turbo87 I've just done in the last commit. Please let me know if that's a fatal error and `init` should be really considered as a lifecycle hook

Reply:
At least api docs suggests there is no init() event/hook in the Component https://emberjs.com/api/ember/3.0/classes/Component/events